infra/base-images/base-builder/Dockerfile

```diff
@@ -144,6 +144,7 @@ COPY cargo compile compile_afl compile_libfuzzer compile_honggfuzz \
     compile_go_fuzzer \
     compile_native_go_fuzzer \
     compile_python_fuzzer \
+    compile_gfuzztests.sh \
```

Comment:
why  plural?

Reply:
Because one call to `compile_gfuzztests.sh` compiles all fuzztest fuzzers, whereas the other scripts each compile a single fuzzer